app/src/test/java/app/RootControllerTest.java

```diff
@@ -796,6 +798,118 @@ public void theCSVFileShouldNotContainCellsBeginningWithUnsafeCharacters() throw
         }
     }
 
+    @Test
+    public void jurisdictionAdminCannotCreateAJurisdiction() {
```

Comment:
Thoughts on moving these tests into `TenantAdminControllerTest.java`?

Reply:
Yes, I'd prefer to have separate tests per controller and would like to introduce one of `JurisdictionController` as well. I've had issues doing so in the past related to boostrapped data. Once we have flyway and remove h2, I think it would be easer to add separate tests.

Reply:
Cool sounds good :+1: